src/uu/kill/src/kill.rs

```diff
@@ -72,7 +77,7 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
         Mode::List => list(pids_or_signals.get(0).cloned()),
     }
 
-    EXIT_OK
+    Ok(())
```

Comment:
Can `EXIT_OK` and `EXIT_ERR` be removed?

Reply:
f0408d5d4acbe1999f9cd1189d9457fa6a67d5e8 fixes that :)